packages/react-native/README.md

```diff
@@ -67,7 +83,7 @@ const YourAppLayout = () => {
     notificationFeed.fetch();
   }, [notificationFeed]);
 
-  return <span>Total unread: {metadata.unread_count}</span>;
+  return <View>Total unread: {metadata.unread_count}</View>;
```

Comment:
Text will need to be wrapped in a `react-native` `<Text/>` component, just putting it in a view will cause it to error.

Reply:
Ahh nice catch, missed that in the readme